app/navigation/ReturnsJourneyNavigator.scala

```diff
@@ -19,14 +19,22 @@ package navigation
 import controllers.returns.routes
 import models.{CheckMode, Mode, NormalMode, UserAnswers}
 import pages._
-import pages.returns.{ConvertedPackagingCreditPage, DirectlyExportedComponentsPage, ExportedPlasticPackagingWeightPage, HumanMedicinesPlasticPackagingPage, HumanMedicinesPlasticPackagingWeightPage, ImportedPlasticPackagingPage, ImportedPlasticPackagingWeightPage, ManufacturedPlasticPackagingPage, ManufacturedPlasticPackagingWeightPage, RecycledPlasticPackagingWeightPage, StartYourReturnPage}
+import pages.returns._
 import play.api.mvc.Call
 
 import javax.inject.Singleton
 
 @Singleton
 class ReturnsJourneyNavigator {
 
+  private def exportedAllPlastic(answers: UserAnswers): Boolean = {
+    val manufactured = answers.get(ManufacturedPlasticPackagingWeightPage).getOrElse(0L)
+    val imported     = answers.get(ImportedPlasticPackagingWeightPage).getOrElse(0L)
+    val exported     = answers.get(ExportedPlasticPackagingWeightPage).getOrElse(0L)
+
+    exported >= (manufactured + imported)
+  }
+
```

Comment:
Here the same a previously:
I wonder if in this case, where the question was not answered we should redirect the user to the home account page where it can start the journey again. In this situation probably something went wrong (because we cannot find the ManufacturedPlasticPackagingWeightPage, ImportedPlasticPackagingWeightPage and ExportedPlasticPackagingWeightPagein the json). Just thinking at the case that happened in the past where user navigate to the page using URL (maybe they had the page cached).

I think also that some of the pages work this way too.

Also can be exportedAllPlastic be exported and re-used. I think we have the same function in different part of the code